services/gr-css-reports/setup.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+set -e
+set -x
+set -o pipefail
+
+# Install taskcluster CLI
+cd "${0%/*}"
+./taskcluster.sh
+
+# shellcheck source=recipes/linux/common.sh
+source "${0%/*}/common.sh"
+
+# Install fuzzfetch
+pip3 install fuzzfetch
```

Comment:
This should already be installed in the grizzly base. (`setup.sh` calls the `fuzzfetch.sh` recipe).